db/seeds.rb

```diff
@@ -1,7 +1,59 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+puts "Creating 2 unique users"
+2.times do
+  User.create({
+    email: Faker::Internet.email,
+    password: '123456',
+    username: Faker::GameOfThrones.dragon
+  })
+end
+
+puts "Created 2 unique users"
+
+#_______________________________________________________________________________
+puts "Creating 4 real businesses"
+
+Business.create({
+  name: "Starbucks",
+  address: " R. Voluntários da Pátria, 89 - Botafogo, Rio de Janeiro - RJ",
+  average_rating: [1, 2, 3, 4, 5].sample,
+  average_english_rating: [1, 2, 3, 4, 5].sample,
+  photo: "https://picsum.photos/200/300/?random"
+})
+Business.create({
+  name: "WeWork",
+  address: "495 Visconde de Piraja Avenue, Rio de Janeiro - RJ",
+  average_rating: [1, 2, 3, 4, 5].sample,
+  average_english_rating: [1, 2, 3, 4, 5].sample,
+  photo: "https://picsum.photos/200/300/?random"
+})
+Business.create({
+  name: "Armazém do café",
+  address: "R. Visc. de Pirajá, 261, Rio de Janeiro",
+  average_rating: [1, 2, 3, 4, 5].sample,
+  average_english_rating: [1, 2, 3, 4, 5].sample,
+  photo: "https://picsum.photos/200/300/?random"
+})
+Business.create({
+  name: "Kilograma",
+  address: "R. Visc. de Pirajá, 644,  Ipanema, Rio de Janeiro - RJ",
+  average_rating: [1, 2, 3, 4, 5].sample,
+  average_english_rating: [1, 2, 3, 4, 5].sample,
+  photo: "https://picsum.photos/200/300/?random"
+})
+
+
+puts "Created 4 real businesses"
+
+#_______________________________________________________________________________
+puts "Creating 10 unique reviews"
+10.times do
+  Review.create({
+    description: Faker::GameOfThrones.quote,
+    english_rating: [1, 2, 3, 4, 5].sample,
+    rating: [1, 2, 3, 4, 5].sample,
+    user_id: [User.first.id, User.last.id].sample,
```

Comment:
As a general comment, do we want more seeds than this?  
If so, this line should copy the line below - User.ids.sample.
Then we are not limited to just 2 users should we add more to the seed... just future proofing.

Reply:
I do not think think so for not, but I have changed it to User.ids.sample